src/stories/2-frontend-engine/frontend-engine.stories.tsx

```diff
@@ -20,11 +20,8 @@ const meta: Meta = {
 			page: () => (
 				<>
 					<Title>FrontendEngine</Title>
-					<Description>
-						The main component to render a form, based on a JSON schema through the `data` prop.
-					</Description>
-					<Heading>Props</Heading>
-					<ArgsTable story={PRIMARY_STORY} />
+					<p>The main component to render a form, based on a JSON schema through the `data` prop.</p>
```

Comment:
Description also applied markdown styling, now the text is getting rendered as plain `` `data` ``

from the docs, this can be done

```
page: () => (
  <>
     <Title>Test</Title>
     <Description />
      //...
  <>,
description: {
  component: "The main component to render a form, based on a JSON schema through the `data` prop."
}
```

Reply:
this didnt work for me so I used the code tag instead